executor/builder.go

```diff
@@ -1476,8 +1489,35 @@ func (b *executorBuilder) buildTableDual(v *plannercore.PhysicalTableDual) Execu
 	return e
 }
 
-// `getSnapshotTS` returns the timestamp of the snapshot that a reader should read.
+// `getSnapshotTS` returns for-update-ts if in insert/update/delete/lock statement otherwise snapshot-read-ts
```

Comment:
I think maybe you can mention that under RC isolation level, the *for-update_ts* and the *snapshot-read-ts* are the same.